### PR TITLE
测试01

### PR DESCRIPTION
**Pull request by mlincyy**
测试测试

### CODE REVIEW

**Comment by mlincyy** [comment]
nothing

**Comment by sandy081** [comment]
Not explained what the PR does and what it is going to fix. Please provide this information and reopen it.

File: .eslintplugin/code-no-nls-in-standalone-editor.ts [hunk 1]
```diff
@@ -21,9 +21,9 @@ export = new class NoNlsInStandaloneEditorRule implements eslint.Rule.RuleModule
 		if (
 			/vs(\/|\\)editor(\/|\\)standalone(\/|\\)/.test(fileName)
 			|| /vs(\/|\\)editor(\/|\\)common(\/|\\)standalone(\/|\\)/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.api/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.main/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.worker/.test(fileName)
+			// || /vs(\/|\\)editor(\/|\\)editor.api/.test(fileName)
```
Comment (huajiankan) [inline .eslintplugin/code-no-nls-in-standalone-editor.ts:24]: 测试 1

File: .eslintplugin/code-no-nls-in-standalone-editor.ts [hunk 2]
```diff
@@ -21,9 +21,9 @@ export = new class NoNlsInStandaloneEditorRule implements eslint.Rule.RuleModule
...
+			// || /vs(\/|\\)editor(\/|\\)editor.main/.test(fileName)
+			// || /vs(\/|\\)editor(\/|\\)editor.worker/.test(fileName)
```
Comment (huajiankan) [inline .eslintplugin/code-no-nls-in-standalone-editor.ts:26]: 测试 2

File: .eslintplugin/code-no-nls-in-standalone-editor.ts [hunk 3]
```diff
@@ -21,9 +21,9 @@ export = new class NoNlsInStandaloneEditorRule implements eslint.Rule.RuleModule
 		if (
 			/vs(\/|\\)editor(\/|\\)standalone(\/|\\)/.test(fileName)
 			|| /vs(\/|\\)editor(\/|\\)common(\/|\\)standalone(\/|\\)/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.api/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.main/.test(fileName)
-			|| /vs(\/|\\)editor(\/|\\)editor.worker/.test(fileName)
```
Comment (huajiankan) [inline .eslintplugin/code-no-nls-in-standalone-editor.ts:26]: 测试 3

**Pull request closed without merging**